tests/contrib/langchain/conftest.py

```diff
@@ -1,73 +1,131 @@
+from http.server import BaseHTTPRequestHandler
+from http.server import HTTPServer
 import os
+import threading
 
-import mock
 import pytest
 
 from ddtrace.contrib.internal.langchain.patch import patch
 from ddtrace.contrib.internal.langchain.patch import unpatch
+from ddtrace.llmobs import LLMObs as llmobs_service
+from ddtrace.llmobs._writer import LLMObsSpanWriter
 from ddtrace.trace import Pin
 from tests.utils import DummyTracer
-from tests.utils import DummyWriter
-from tests.utils import override_config
 from tests.utils import override_env
 from tests.utils import override_global_config
 
 
+class TestLLMObsSpanWriter(LLMObsSpanWriter):
```

Comment:
I think we can use the common util found in `tests/llmobs/_utils.py`

Reply:
yes! i need to rebase the PR but i'll use the shared utils you added